dev/source/docs/making-a-mavlink-wifi-bridge-using-the-raspberry-pi.rst

```diff
@@ -416,22 +416,41 @@ Update hostapd
 --------------
 
 The hostapd version installed by apt-get does not fully support the WiFi
-dongle we are using so we need to update it to a later version. Lets get
-the new version of hostapd by typing the follwing command:
+dongle we are using so we need to update it to a later version. 
+
+To get the new version of hostapd for the Adafruit Dongle type the following command:
 
 ::
 
     wget http://adafruit-download.s3.amazonaws.com/adafruit_hostapd_14128.zip
+    
+To get the new version of hostapd for the Edimax 7811 Dongle type the following command:
+
+::
+
+    wget http://www.daveconroy.com/wp3/wp-content/uploads/2013/07/hostapd.zip
```

Comment:
I followed this tutorial to get it going: http://www.daveconroy.com/turn-your-raspberry-pi-into-a-wifi-hotspot-with-edimax-nano-usb-ew-7811un-rtl8188cus-chipset/

Perhaps there is a way to host his zip file on the JDrone Servers?